src/style.css

```diff
@@ -0,0 +1,141 @@
+body {
+  margin: 0;
+  font-family: "Roboto", Helvetica, sans-serif;
+}
+
+.card {
+  width: 200px;
+  box-sizing: border-box;
```

Comment:
better to set `box-sizing` for all elements using `*` selector

`* { box-sizing: border-box; }`